server/controllers/Contacts.js

```diff
@@ -12,6 +13,7 @@ module.exports.addContact = async (req, res) => {
     // Validate request (need to add check to ensure user exists) +
     // (prevent user from adding self as contact????)
     if (!req.body.contactEmail) {
+        logger.error("Cannot add a contact without their email");
         res.status(400).send({
             msg: "Content can not be empty!"
```

Comment:
Maybe update the msg to match the logger message. The Style used later (define a msg variable then use it in logger.error and res.send) would be better?